app/safe/internal/bootstrap/bootstrap.go

```diff
@@ -188,7 +188,7 @@ func CreateCryptoKey(id *string, updatedSecret chan<- bool) {
 
 	log.InfoLn(id, "Generated public key, private key, and aes seed")
 
-	if err = persistKeys(privateKey, publicKey, aesSeed); err != nil {
+	if err = PersistKeys(privateKey, publicKey, aesSeed); err != nil {
```

Comment:
What’s the reasoning behind making this method public?

From the looks of it, it’s only used in this module.

Is there a specific need to keep it public?

Reply:
[ReceiveKeys()](https://github.com/vmware-tanzu/secrets-manager/pull/564/files/3dcc0f8086cf86e2185778adacdc4c4206f6df42#diff-722ad5277a6975ab286b63b74eddb3809114bd3efc157ebb33a2160f48b6f0d6R27) function of the Safe saves the input keys and I aim to perform the [persistKeys()](https://github.com/vmware-tanzu/secrets-manager/pull/564/files/3dcc0f8086cf86e2185778adacdc4c4206f6df42#diff-722ad5277a6975ab286b63b74eddb3809114bd3efc157ebb33a2160f48b6f0d6R71) operations in that function if the relative env variable is set. To avoid duplication, I planned to export the private persistKeys() from the bootstrap module to call it from the /server/route/receive.go file.

Reply:
Noted. Fair enough. — Let’s see how it unfolds :)